samples/gameSample/src/tui.cpp

```diff
@@ -29,9 +29,9 @@ int TUI::scanKeys()
     return VK_NONAME;
 }
 
-void TUI::setPosition(point_t position)
+void TUI::SetCursor(Point position)
 {
-    std::cout << "\033[" << position.x << ";" << position.y << "H";
+    std::cout << "\033[" << position.y << ";" << position.x << "H";

```

Comment:
https://learn.microsoft.com/en-us/windows/console/setconsolecursorposition
They seem to have APIs for this as well.